core/physical_substances.mcf

```diff
@@ -0,0 +1,114 @@
+# Schema for all chemical elements and compounds
+
+Node: dcid:PhysicalSubstance
+typeOf: schema:Class
+name: Physical Substance
+description: All types of substances found on earth like elements, compounds belonging to all three states of matter (solid, liquid and gas)
+
+Node: dcid:Element
+typeOf: schema:Class
+name: Element
+subClassOf: dcs:PhysicalSubstance
+description: Elements in the periodic table
+
+Node: dcid:Compound
+typeOf: schema:Class
+name: Compound
+subClassOf: dcs:PhysicalSubstance
+description: Compounds are physical substances that contain more than one element. Compounds can be homogeneous (E.g.: O2) or heterogeneous (E.g.: H2O).
+
+
+## Nodes for elements and commonly known compounds
+## More properties like atomic number and chemical characteristics can be added for the node
+
+# Elements
+
+Node: dcs:Aluminium
+typeOf: dcs:Element
+name: "Aluminium"
+
+Node: dcs:Arsenic
+typeOf: dcs:Element
+name: "Arsenic"
+
+Node: dcs:Calcium
+typeOf: dcs:Element
+name: "Calcium"
+
+Node: dcs:Chlorine
+typeOf: dcs:Element
+name: "Chlorine"
+
+Node: dcs:Fluorine
+typeOf: dcs:Element
+name: "Fluorine"
+
+Node: dcs:Iron
+typeOf: dcs:Element
+name: "Iron"
+
+Node: dcs:Lead
+typeOf: dcs:Element
+name: "Lead"
+
+Node: dcs:Potassium
+typeOf: dcs:Element
+name: "Potassium"
+
+Node: dcs:Magnesium
+typeOf: dcs:Element
+name: "Magnesium"
+
+Node: dcs:Mercury
+typeOf: dcs:Element
+name: "Mercury"
+
+Node: dcs:Phosphate
+typeOf: dcs:Element
+name: "Phosphate"
+
+Node: dcs:Sodium
+typeOf: dcs:Element
+name: "Sodium"
+
+Node: dcs:Silver
+typeOf: dcs:Element
+name: "Silver"
+
+Node: dcs:Zinc
+typeOf: dcs:Element
+name: "Zinc"
+
+# Compounds
+
+Node: dcs:Ammonia
+typeOf: dcs:Compound
+name: "Ammonia"
+
+Node: dcs:Carbonate
+typeOf: dcs:Compound
+name: "Carbonate"
+
+Node: dcs:Cyanides
+typeOf: dcs:Compound
+name: "Cyanides"
+
+Node: dcs:HydrogenCarbonate
+typeOf: dcs:Compound
+name: "HydrogenCarbonate"
+
+Node: dcs:Nitrate
+typeOf: dcs:Compound
+name: "Nitrate"
+
+Node: dcs:Sulfate
+typeOf: dcs:Compound
+name: "Sulfate"
+
+Node: dcs:SiliconDiOxide
+typeOf: dcs:Compound
+name: "Silicon dioxide"
```

Comment:
Can you replace `Node: dcs:<elementName/ compoundName>` with `Node: dcid<elementName/ compoundName>`.

For example: 
- element: `Node: dcs:Sodium` to `Node: dcid:Sodium`
- compound: `Node: dcs:HydrogenCarbonate` to `Node: dcid:HydrogenCarbonate